package.json

```diff
@@ -137,7 +137,6 @@
     "@types/estree": "link:./tools/dummypkg",
     "@types/node": "^20.0.0",
     "@types/react": "^18.2.14",
-    "eslint": "^9",
```

Comment:
Unsure of exactly what we wanna do with the various places the dependency is declared. I removed the `resolutions` field and did `yarn up eslint`. But, lmk if we want to manage it differently